mqtt/mqtt-bridge/tests/config.json

```diff
@@ -27,6 +27,43 @@
                 "topic": "floor2/#",
                 "inPrefix": "",
                 "outPrefix": ""
+            },
```

Comment:
This will conflict with this PR I suppose? https://github.com/Azure/iotedge/pull/4559/files

Reply:
yeap, conflict